### PR TITLE
[MOSTLY MODULAR] Securitrons and ED-209s are now craftable

### PR DESCRIPTION
**Pull request by FlamingLily**
## About The Pull Request
Adds 2 items, the bot part packs, orderable from cargo. They are used as replacement for disablers/stun batons.

makes the bots no longer drop disablers/stun batons, either.

## Why It's Good For The Game
Missing content is no longer missing.

## Changelog
:cl:
add: BOTco security robot flat packs have now been released! Order a securitron or ED-209 part kit from cargo today!
fix: The aforementioned robots are now craftable again, with the part kits.
del: They also no longer drop human-usable stun batons or disablers, on account of not using them in their construction, sorry greytiders, no free weapons for you.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -1336,6 +1336,8 @@
 	access_view = ACCESS_RESEARCH
 	crate_type = /obj/structure/closet/crate/science
 
+/datum/supply_pack/science/ed209pack // SKYRAT EDIT ADDITION
```
Comment (Gandalf2k15) [inline code/modules/cargo/packs.dm]: What

Comment (FlamingLily) [reply code/modules/cargo/packs.dm]: ?

**Comment by tf-4** [comment]
> You currently have a negative Fix/Feature pull request delta of -32. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

Feature coder smh

**Comment by FlamingLily** [comment]
> 
> 
> > You currently have a negative Fix/Feature pull request delta of -32. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.
> 
> Feature coder smh

I am absolutely sure my actual feature/fix ratio is like, 1.5 features for each 1 fix.

Not my fault the bot values fixes at like 1 delta and features at like -40

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**